.tt_blacklist

```diff
@@ -1,6 +1,4 @@
 chemicaltoolbox/osra
-chemicaltoolbox/silicos-it/shape-it
```

Comment:
I would not mix this up. This should be in an other PR. 
Btw. as soon as you remove something from the blacklist, please trigger a test by changing something in the XML file of the corresponding tool - Some whitespace changes, spelling errors and such things. This will trigger then a real test and we see if the testing works.